website/docs/r/mysql_server.html.markdown

```diff
@@ -84,7 +84,7 @@ The following arguments are supported:
 
 `storage_profile` supports the following:
 
-* `storage_mb` - (Required) Max storage allowed for a server, possible values are between `5120 MB` (5GB) and `1048576 MB` (1TB). The step for this value must be in `1024 MB` (1GB) increments. For more information see the [product documentation](https://docs.microsoft.com/en-us/rest/api/mysql/servers/create#StorageProfile).
+* `storage_mb` - (Required) Max storage allowed for a server, possible values are between `5120 MB` (5GB) and `4194304 MB` (4TB). The step for this value must be in `1024 MB` (1GB) increments. For more information see the [product documentation](https://docs.microsoft.com/en-us/rest/api/mysql/servers/create#StorageProfile).
```

Comment:
Can we change this to be: `...possible values are between `5120 MB` (5GB) and `1048576 MB` (1TB) for the Basic SKU and between `5120 MB` (5GB) and `4194304 MB` (4TB) for General Purpose/Memory Optimized SKUs.`?

Reply:
sure, will add that to my commit with the conditional